packages/file_selector/file_selector_web/integration_test/lib/main.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Can you move this to an `example` directory, and then add the integration tests following this documentation?

https://flutter.dev/docs/testing/integration-tests

They describe a cool directory structure there (you can share the driver, and run tests).

Reply:
Done. PTAL